tests/std/tests/P0980R1_constexpr_strings/test.cpp

```diff
@@ -530,16 +530,12 @@ constexpr bool test_interface() {
         literal_constructed.shrink_to_fit();

```

Comment:
FWIW, I think we should also add some test coverage for constexpr that grows from small, shrinks to small, etc. since we're exercising a lot of new code paths here!